testsuite/classlibrary/TestNamedControl.sc

```diff
@@ -0,0 +1,57 @@
+TestNamedControl : UnitTest {
+
+	test_default_value_in_single_instance {
```

Comment:
Following the conventions I recommended, the name of this test would become:

```
test_defaultValue_inSingleInstance_isZero
```

and the message can be removed